utils/urlencode.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015 Spinpunch, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package utils
+
+import (
+	"net/url"
+	"strings"
+)
+
+func UrlEncode(str string) string {
```

Comment:
Those methods are for HTML escaping, aren't they? This is for URL encoding which is different. I also searched to see if an existing url encoder function existed in a go lib but couldn't find one